docs/resources/dws_cluster.md

```diff
@@ -74,6 +74,7 @@ The following arguments are supported:
   Changing this parameter will create a new resource.
 
 * `availability_zone` - (Required, String, ForceNew) The availability zone in which to create the cluster instance.
+  If there are multiple available zones, separate by commas, e.g. **cn-north-11,cn-north-5**.
```

Comment:
Lack of description information, for example:
1、Currently, multi-AZ clusters only support selecting 3 AZs.
2、Only when version has a value, multiple az can be configured.

Reply:
1.done
2.the version is required, so there is no need to pay attention to it.